app/boards/shields/nibble/Kconfig.defconfig

```
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if SHIELD_NIBBLE
+
+config ZMK_KEYBOARD_NAME
+	default "NIBBLE_BLE"
```

This can just be:

```suggestion
	default "NIBBLE"
```
for the keyboard name.